apps/token-manager/contracts/TokenManager.sol

```diff
@@ -99,10 +100,11 @@ contract TokenManager is ITokenController, IForwarder, AragonApp {
 
     /**
     * @notice Mint `@tokenAmount(self.token(): address, _amount, false)` tokens for `_receiver`
-    * @param _receiver The address receiving the tokens
+    * @param _receiver The address receiving the tokens, cannot be the Token Manager itself (use `issue()` instead)
     * @param _amount Number of tokens minted
     */
     function mint(address _receiver, uint256 _amount) external authP(MINT_ROLE, arr(_receiver, _amount)) {
+        require(_receiver != address(this), ERROR_MINT_RECEIVER_IS_TM);
```

Comment:
Totally support doing this. We should update the UI so it takes this change into account and calls the proper function when the TM is inputted in the 'Assign Tokens' panel.

Reply:
See https://github.com/aragon/aragon-apps/issues/748. We could also do a better job of actually showing the Token Manager's balances, if any.